app/uk/gov/hmrc/bindingtariffclassification/model/MongoFormatters.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.model
+
+import java.time.{Instant, ZoneId, ZonedDateTime}
+
+import play.api.libs.json._
+import uk.gov.hmrc.play.json.Union
+
+object MongoFormatters {
```

Comment:
I've pulled these out into two Formatter files as all Mongo Formatters need to render dates as 
```
"someDateField": {"$date": 12345}
```
For Mongo to pick up that it is actually a date.

Reply:
REST payloads obviously dont want this so I've created a RESTFormatters as well. Note there is different formatters in each, some classes e.g. `NewCaseRequest` `NewEventRequest` etc don't need MongoFormatters and classes like `Sequence` done need RESTFormatters

Reply:
We'll need to clear down the DB for this to work though.